src/FrameReflower/Text.php

```diff
@@ -386,13 +389,35 @@ function reflow(BlockFrameDecorator $block = null)
 
         // Determine the text height
         $style = $frame->get_style();
+        $font_metrics = $this->getFontMetrics();
+
+        // Handle text transform and white space
+        $text = $this->pre_process_text($frame->get_text());
+
+        // map text to fonts based on supported Unicode range
+        $families = array_map(
+            function ($value) {
+                return trim($value, " '\"");
+            },
+            explode(",", $style->get_specified("font_family"))
+        );
+        $charMapping = $font_metrics->mapFontsToText($text, $families, $font_metrics->getType($style->font_weight . ' ' . $style->font_style), 1);
```

Comment:
This has not changed, but still wondering here: As `FontMetrics::getType()` is always used with a parameter like `"$weight $style"` and then goes on to split that input again, why does the function not just take 2 parameters?

Reply:
Then again, it's not used in very many places so maybe I will go ahead and make that change.

Reply:
Actually, following through usage and I think I'll put this off for that future change. The way that return value flows through to other method calls indicates that some more thought should be put into it than I want to do now.